app/controllers/api/v1/search_controller.rb

```diff
@@ -317,6 +476,35 @@ def self.match_studies_by_facet(query_results, search_facets)
         end
         matches
       end
+
+      # Helper method for generating a curl command to download a file from a bucket.  Returns file's curl config, size.
```

Comment:
move this method to the BulkDownloadService :)

Reply:
f5a85e5